src/data/compositors/mutter.json

```diff
@@ -1,122 +1,129 @@
 {
-  "generationTimestamp": 1709750464116,
-  "version": "46",
```

Comment:
Why did the version get removed?

Reply:
Is it necessary to add subversion? I.E. 47.3?

Reply:
If supported protocol versions change between minor versions of Mutter it would be worth adding the minor version too. Otherwise just the major version is probably fine.

In any case, it's fine for me either way (but I would prefer to keep the version in even it means manually editing the JSON).